src/app/main-page/main-page.html

```diff
@@ -0,0 +1,40 @@
+<div class="img">
+    <div class="left">
+        <div class="title-container">
+            <div class="title" ui-sref="startPage" ui-sref-active="active">
+                <h1 class="brown">C</h1>
+                <h1 class="salmon">a</h1>
+                <h1 class="skyblue">e</h1>
+                <h1 class="brown">l</h1>
+                <h1 class="salmon">u</h1>
+                <h1 class="skyblue">m</h1>
+                <h1 class="brown">W</h1>
+                <h1 class="salmon">e</h1>
+                <h1 class="skyblue">a</h1>
+                <h1 class="brown">t</h1>
+                <h1 class="salmon">h</h1>
+                <h1 class="skyblue">e</h1>
+                <h1 class="brown">r</h1>
+                <a class="reg salmon">™</a>
+            </div>
+        </div>
+        <div class="form-container">
+            <form autocomplete="off" layout="row" layout-align="center center" ng-submit="$ctrl.searchCity($ctrl.city)">
+                <md-input-container class="input-container">
+                    <label>{{isSpanish ? 'Mostrarme el tiempo en...' : 'Show me the weather in...'}}</label>
```

Comment:
If you want your app to be localized you should use angular translate https://github.com/angular-translate. Otherwise stick with one language.

Reply:
I had planned to use angular-translate in the next sprint